rafthttp/transport.go

```diff
@@ -180,7 +180,10 @@ func (t *Transport) Send(msgs []raftpb.Message) {
 		}
 		to := types.ID(m.To)
 
+		t.mu.RLock()
 		p, ok := t.peers[to]
+		t.mu.RUnlock()
```

Comment:
`mu` is used to `protect the remote and peer map`, and i see `g, ok := t.remotes[to]` 10  lines below, so I think it needs this lock too.